src/main/java/net/openhft/chronicle/wire/WireMarshaller.java

```diff
@@ -753,14 +756,15 @@ protected void setValue(Object o, @NotNull ValueIn read, boolean overwrite) thro
                             !type.isEnum() &&
                             !read.isTyped()) {
                         // retain the null value of object
+                        Jvm.warn().on(getClass(), "Ignoring exception and setting field '" + field.getName() + "' to null", e);
```

Comment:
It might be nice to log a warning when we swallow this exception? it seems very broad and made this problem harder to spot

Reply:
Agree